cmd/storage-rpc-client.go

```diff
@@ -32,6 +33,15 @@ type networkStorage struct {
 	connected bool
 }
 
+// List of Call timeouts for each class of storage operations
+const (
+	volumeOpTimeout        = 10 * time.Second
+	volumeListOpTimeout    = 10 * time.Minute
+	fileStatOpTimeout      = 10 * time.Second
+	fileReadWriteOpTimeout = 10 * time.Minute
+	fileListOpTimeout      = 30 * time.Minute
```

Comment:
@harshavardhana, I think having large timeout values doesn't fix the problem of clients wondering why it takes too much time to list a bucket contents when one node fails. I think to fix this use case, we need to fork net/rpc and we see why even when KeepAlive timeout is set, rpc.Call() doesn't quit when one node fails.

Reply:
The issue is actually not in rpc.Call() @vadmeste the issue is basically in net.Conn which was previously remembered doesn't return or timeout. Without deadlines Go has no timeouts for net.Conn